scripts/database/delete_unfixable_orcids.sql

```diff
@@ -0,0 +1,53 @@
+-- !!!!!!!  run fix_fixable_orcids.sql before this script,
+-- !!!!!!!  otherwse fixable ORCIDs will be lost
+DO $$
+DECLARE
+    ids numeric[];
+BEGIN
+	-- find indetifiers of all dataset fields with supposed ORCID values 
+	-- that don't match ORCIS format
+    SELECT ARRAY(select id
+        from datasetfield 
+         where 
+         datasetfieldtype_id in (select id 
+                        from datasetfieldtype 
+                        where 
+                        name = 'authorIdentifier')  
+        and
+        datasetfieldparent_id in 
+            (select datasetfieldparent_id 
+             from datasetfield  
+             where 
+                id in (select datasetfield_id 
```

Comment:
You need also a check that the field is of type `authorIdentifierScheme`.

Reply:
fixed